Mathlib/Algebra/Module/Injective.lean

```diff
@@ -96,7 +94,7 @@ set_option linter.uppercaseLean3 false in
 
 namespace Module.Baer
 
-variable {R Q} {M N : Type max u v} [AddCommGroup M] [AddCommGroup N]
+variable {R Q} {M N : Type v} [AddCommGroup M] [AddCommGroup N]
```

Comment:
I think the proofs should just work, but let me know if not.
```suggestion
variable {R Q} {M N : Type*} [AddCommGroup M] [AddCommGroup N]
```

Reply:
This works Thanks